src/it/scala/io/scalac/tezos/translator/MessageSpec.scala

```diff
@@ -24,8 +24,9 @@ class MessageSpec
   extends WordSpec
   with Matchers
   with ScalatestRouteTest
-  with BeforeAndAfterEach
-  with JsonHelper {
+  with BeforeAndAfterEach {
+    import de.heikoseeberger.akkahttpcirce.FailFastCirceSupport._
```

Comment:
Why you import this thing here ?

Reply:
It complains about lack of marshaller for `Post`s there